### PR TITLE
Switch to the Gregorian calendar year

### PR DESCRIPTION
**Pull request by dave1010**
The official ISO 8601 format uses the Gregorian calendar, rather than the more empathic I-feel-like-it's-X calendar. Whilst I prefer the latter, it somewhat rejects the axiom that time will advance without permission.

That whilst the most prolific way to denote the year at the moment seems to be " ̶2̶0̶2̶1̶  2022", I believe the more terse "2021" is best suited for an API.

Note: we had some business critical production systems fail until we manually ssh'd to some servers and fixed this. I'd expect better from an Open Source project. Over 200 Github stars implies some kind of SLA.

### CODE REVIEW

File: currentYear [hunk 1]
```diff
@@ -1 +1 @@
-2021
+2022
```
Comment (asgrim) [inline currentYear:1]: The changes need to be propagated to `nextYear`, `previousYear`, as well as `en/*` too please

Comment (dave1010) [reply currentYear:1]: Done.

I left the leap year thing alone but feel like we should add it in as a bank holiday.

Comment (dave1010) [reply currentYear:1]: Oh, and happy new year!

**Comment by asgrim** [comment]
Thank you @dave1010 for your incredible contribution to open source, and the Year community. You truly are a driving force in the world, and I wish you happiness and prosperity in all your endeavours. Happy Easter!

**Comment by dave1010** [comment]
Thanks @asgrim! 

christmas

**Pull request merged**